### PR TITLE
Update varify to leverage varify-data-warehouse

### PR DESCRIPTION
**Pull request by naegelyd**
Signed-off-by: Don Naegely naegelyd@gmail.com

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -9,27 +9,32 @@
...
-    'tests.cases.commands',
+    'tests.cases.resources',
 
     'varify',
 
-    'varify.raw',
-    'varify.raw.sources',
-
     'varify.assessments',
+    'varify.genes',
```
Comment (bruth) [inline tests/settings.py]: Apps that have the same last import name (e.g. `genes`) will cause all but the first to be acknowledged. I see that with `assessments`, `genes`, `phenotypes`, `samples`, `genome`, and `variants`.

**Comment by bruth** [comment]
@naegelyd It appears some of the tests [have errored](https://travis-ci.org/cbmi/varify/jobs/31355843#L1133). That may be why the coverage drop 10%.

**Comment by naegelyd** [comment]
@bruth I assume you are talking about the `RequestException`s, `SSLError`s, etc. Those are all part of the gene rank test case and are intentionally triggered and expected in the test itself. All 11 tests run successfully. The reason for the decrease in coverage is a result of my reshuffling of some code and tests between varify and vdw and force pushing the changes onto this branch.

**Comment by bruth** [comment]
The errors are value-related, e.g. `ValueError: invalid literal for int() with base 10: 'BAD_RANK'`

**Comment by naegelyd** [comment]
All those errors are expected. See [here](https://github.com/cbmi/varify/blob/master/tests/cases/commands/tests.py#L483-L506) for the `ValueError` generating test case.

**Comment by bruth** [comment]
Ok.

**Pull request merged**